cocos/core/renderer/scene/model.ts

```diff
@@ -158,14 +157,6 @@ export class Model {
         return this._updateStamp;
     }
 
-    /**
-     * @en Whether GPU instancing is enabled for the current model
-     * @zh 是否开启实例化渲染
-     */
-    get isInstancingEnabled () {
-        return this._instMatWorldIdx >= 0;
-    }
```

Comment:
Will it break compatibility.

Reply:
this function is not accuracy enough,  only used by planar-shadow-queue